tests/qa_automation/qa_run.pm

```diff
@@ -17,33 +17,30 @@ use warnings;
 use File::Basename;
 use base "opensusebasetest";
 use utils;
-use testapi;
+use testapi qw(is_serial_terminal :DEFAULT);
 
 sub test_run_list {
     return ();
 }
 
-sub junit_type {
-    die "you need to overload junit_type in your class";
-}
-
-sub test_suite {
-    die "you need to overload test_suite in your class";
-}
-
-
```

Comment:
All these methods do is returning a string. I've modifed the module, and these strings are no longer needed.